Client/Frontend/Browser/HomePanel/BraveShieldStatsView.swift

```diff
@@ -103,9 +103,16 @@ class BraveShieldStatsView: UIView, Themeable {
                 text = Strings.ShieldsTimeStatsDays
             }
             
-            return "\(Int(counter))\(text)"
+            return toLocaleString(num: Int(counter)) + text
         }
     }
+    
+    private func toLocaleString(num: Int) -> String {
```

Comment:
Let's create a `BinaryInteger` (or even just an `Int`) extension here can even be private inside of this file.

```
extension BinaryInteger {
  func localeDisplayString() {}
}```